lib/mutant/ast/named_children.rb

```diff
@@ -13,8 +13,6 @@ module NamedChildren
       # @api private
       #
       def self.included(host)
-        super
```

Comment:
The main reason we used `super` before is if `Module#included` defined some functionality. I realize it doesn't (usually, unless you are using something crazy like AS) but it could.

Overriding the `.included` method is no different from overriding some other "hook" method like `#initialize`. Should we assume when we are overriding that we *know* the precise code in the parent method and make calling `super` optional when it is not needed?